implementations/micrometer-registry-elastic/src/main/java/io/micrometer/elastic/ElasticConfig.java

```diff
@@ -68,12 +68,12 @@ default String host() {
 
     /**
      * The index name to write metrics to.
-     * Default is: "metrics"
+     * Default is: "micrometer-metrics"
      *
      * @return index name
      */
     default String index() {
-        return getString(this, "index").orElse("metrics");
+        return getString(this, "index").orElse("micrometer-metrics");
```

Comment:
I wonder what % of users are not affected by the issue, which with my current understanding would be those using the OSS version of Elasticsearch. This also seems like it will be a pain for all users upgrading because old data will be in one index pattern and new data will be in another.

I like the simplicity of this solution but I wonder if we shouldn't do something more complicated to avoid a painful upgrade unless necessary. We could use a different index prefix only if we detect a version of Elasticsearch affected by the problem. Either way we will need to write good release notes for this.

Reply:
> We could use a different index prefix only if we detect a version of Elasticsearch affected by the problem.

@shakuzen Thanks for the feedback! I thought the same at first, but I inclined to have the same index name prefix in a single Micrometer release for consistency later. I'm not sure which one is better for users though.